main_spec.tex

```diff
@@ -106,16 +106,6 @@ \subsubsection{\textbf{SHMEM\_MALLOC, SHMEM\_FREE, SHMEM\_REALLOC, SHMEM\_ALIGN}
 \subsubsection{\textbf{SHMEM\_CALLOC}}\label{subsec:shmem_calloc}
 \input{content/shmem_calloc.tex}
 
-\subsubsection{\textbf{SHPALLOC}}\label{subsec:shpalloc}
-\input{content/shpalloc.tex}
-
-\subsubsection{\textbf{SHPCLMOVE}}\label{subsec:shpclmove}
-\input{content/shpclmove.tex}
-
-\subsubsection{\textbf{SHPDEALLC}}\label{subsec:shpdeallc}
-\input{content/shpdeallc.tex}
```

Comment:
Please also delete these files.

Reply:
I suggest committing the file deletions (those above and the examples) before making any of the other changes.  It will make the special ballot diff a bit easier to generate.

Reply:
Deleted